ReSharper.FSharp/src/FSharp.Psi.Services/src/Daemon/Highlightings/FcsErrors.xml

```diff
@@ -669,12 +669,12 @@
     <Behavour overlapResolvePolicy="NONE"/>
   </Warning>
 
-  <Error staticGroup="FSharpErrors" name="UnionCaseDoesNotTakeArguments" ID="FS0725: This union case does not take arguments">
+  <Error staticGroup="FSharpErrors" name="RemovePatternArgument" ID="FS0725: This pattern does not take arguments">
```

Comment:
`RemovePatternArgument` is not suitable name for an error, it sounds like an action. Please keep the error names from the compiler, loop them up in FSComp.txt.

Reply:
Ohh I see . I was trying to reuse the same for both . But it seems that I llueve need to add a new error 😅

Reply:
Done